File: src/Bpo.Dms.WebApp/Data/Product.cs
namespace Bpo.Dms.WebApp.Data
{
    public class Product
    {
        public int Id { get; set; }
        public string Code { get; set; }
        public string Name { get; set; }
        public ProductType ProductType { get; set; }
        public string Description { get; set; }
        public decimal Price { get; set; }
        public DateTime ImportDate { get; set; }
    }
}

File: src/Bpo.Dms.WebApp/ClientApp/src/services/models/Product.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { ProductType } from './ProductType';

export type Product = {
    id?: number;
    code?: string | null;
    name?: string | null;
    productType?: ProductType;
    description?: string | null;
    price?: number;
    importDate?: string;
};